<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>459</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Vuelos App</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
    MainWindow {
        background-image: url(&quot;imagenes/fondo.jpg&quot;) background-position: center;
    }
   </string>
  </property>
  <widget class="QWidget" name="principal">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#principal{
background-image: url(:/icons/recursos/media/fondo.jpg);
background-repeat: no-repeat;
background-position: center;
background-attachment: fixed;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Titulo">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>Bienvenido a Vuelos App</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="ContainerLoginYResgister" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>90</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#ContainerLoginYResgister{
min-height: 325;
min-width: 556px;
}
QFrame {
background-color: rgba(255, 255, 255, 150);
border-radius: 15px;
}
QLabel{ 
min-height: 35px;
max-height: 35px;
min-width: 40px;
max-width: 150px;
color:black;
}   
QLineEdit { 
min-height: 35px;
max-height: 35px;
max-width: 150px;
}
QLineEdit{
color:black;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>50</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QWidget" name="widget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>42</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>REGISTRARSE</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="la_nombre">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Nombre: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="li_nombre">
                 <property name="styleSheet">
                  <string notr="true">background-color: white; border-radius: 5px;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Introduce tu usuario</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="la_nombre_3">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Apellido: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="li_apellido">
                 <property name="styleSheet">
                  <string notr="true">background-color: white; border-radius: 5px;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Introduce tu usuario</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="la_contra_3">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Contraseña:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="li_contra_2">
                 <property name="styleSheet">
                  <string notr="true">background-color: white; border-radius: 5px;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Introduce tu usuario</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="la_usu_3">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Email:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="li_usuario_2">
                 <property name="styleSheet">
                  <string notr="true">background-color: white; border-radius: 5px;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Introduce tu usuario</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="la_nombre_2">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>DNI:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="li_dni">
                 <property name="styleSheet">
                  <string notr="true">background-color: white; border-radius: 5px;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Introduce tu usuario</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="bu_reg">
               <property name="styleSheet">
                <string notr="true">
        QPushButton {
            background-color: #2ecc71;
            color: white;
            border-radius: 5px;
        }
        QPushButton:hover {
            background-color: #27ae60;
        }
      </string>
               </property>
               <property name="text">
                <string>Registrarse</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>50</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QWidget" name="horizontalWidget_5" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>LOGIN</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="verticalWidget_2" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="la_usu">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Email:</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::TextFormat::AutoText</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="li_usuario">
                  <property name="styleSheet">
                   <string notr="true">background-color: white; border-radius: 5px;</string>
                  </property>
                  <property name="placeholderText">
                   <string>Introduce tu usuario</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="horizontalWidget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="la_contra">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Contraseña:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="li_contra">
                  <property name="styleSheet">
                   <string notr="true">background-color: white; border-radius: 5px;</string>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::EchoMode::Password</enum>
                  </property>
                  <property name="placeholderText">
                   <string>Introduce tu contraseña</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bu_login">
               <property name="styleSheet">
                <string notr="true">
        QPushButton {
            background-color: #3498db;
            color: white;
            border-radius: 5px;
        }
        QPushButton:hover {
            background-color: #2980b9;
        }
      </string>
               </property>
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QPushButton" name="bu_salir">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
        QPushButton {
            background-color: red;
            color: white;
            border-radius: 5px;
        }
        QPushButton:hover {
            background-color: darkred;
        }
      </string>
            </property>
            <property name="text">
             <string>Salir</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
